config-native/build.gradle

```diff
@@ -23,23 +23,9 @@ description = 'Ballerina - Config Java Utils'
 
 dependencies {
     compile group: 'org.ballerinalang', name: 'ballerina-lang', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-config', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-reflect', version: "${ballerinaLangVersion}"
     compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-stringutils', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-runtime-api', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-cache', version: "${ballerinaLangVersion}"
     compile group: 'org.slf4j', name: 'slf4j-jdk14', version: "${slf4jVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-tool', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-java', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-time', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-system', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'ballerina-io', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'value', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'typedesc', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'annotations', version: "${ballerinaLangVersion}"
-    compile group: 'org.ballerinalang', name: 'toml-parser', version: "${ballerinaTomlParserVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-config', version: "${ballerinaLangVersion}"
```

Comment:
Any reason to have config module as dependency on itself?

Reply:
`ConfigRegistry` has not been moved to the module yet. Therefore, the build fails without this dependecny.